test/repo/Makefile

```diff
@@ -81,3 +82,15 @@ compare:
 	@ for html in $$(cd $(REPORT_ROOT2) && find . -name html); do \
 	  diff -rq $(REPORT_ROOT1)/$$html $(REPORT_ROOT2)/$$html; \
 	done
+
+summarize:
```

Comment:
What is this actually testing? I'm not sure that I understand. This just seems to be running the script with the flag?

Reply:
Yes, it's indeed just running the script using the new flag.

I added this target, because it served as a means for me to illustrate this new output without having to ask a reviewer/user to run anything on their machine.

Reply:
I will be removing this target since the Makefile already had a specific intent. I should instead create a separate test.